test/algorithms/classifiers/test_qsvc.py

```diff
@@ -101,6 +101,19 @@ def test_qsvc_parameters(self):
 
         self.assertEqual(score, 0.5)
 
+    def test_args_warning(self):
+        """Test QSVC deprecation warning appears when *args passed in"""
+        qsvc = QSVC(1)
+
+        with self.assertWarns(DeprecationWarning):
```

Comment:
Isn't the deprecation warning issued when the class in constructed? Now it maybe that since we are passing in args that it is failing in this line like the issue reported - not sure whether we want to test it failing. For sure passing no args it should convert to str, so would print, so checking that the issue no longer exists in that case seems fine.